lib/ui/app.js

```diff
@@ -50,6 +52,9 @@ export default class App {
     this.socket.on('game list', (data) => {
       this.lobby.update(data.games);
     });
+
+    this.socket.on('session event', data => this.handleSessionEvent(data));
```

Comment:
You get the wrong binding that way. Over the arrow notation I don't see much aesthetic merit to
```this.socket.on('session event', this.handleSessionEvent.bind(this));```